.renovaterc.json5

```diff
@@ -13,15 +13,15 @@
         "xnatdev/xnat-web",
         "xnatdev/xsync",
         "xnatx/ldap-auth-plugin",
-        "xnatdev/container-service",
-        "xnatx/xnatx-batch-launch-plugin",
-        "VUIIS/dax",
         "icrimaginginformatics/ohif-viewer-xnat-plugin",
         "xnatx/ml-plugin",
         "xnatx/datasets-plugin",
         "xnatdev/xnat-image-viewer-plugin",
         "xnatx/xnatx-dxm-settings-plugin",
         "NrgXnat/xnat-pipeline-engine",
+        "xnatdev/container-service",
+        "xnatx/xnatx-batch-launch-plugin",
+        "VUIIS/dax",
```

Comment:
I moved these to the bottom as they're no longer in the default list of plugins (they're only used in testing with molecule)